services/viva-ms/serverless.yml

```diff
@@ -84,6 +85,39 @@ provider:
           Resource: !Sub arn:aws:kms:${AWS::Region}:${AWS::AccountId}:key/*
 
 functions:
+  personApplicationStatus:
+    handler: lambdas/personApplicationStatus.main
+    timeout: 15
```

Comment:
Whats the execution time of the lambda function?

Reply:
Default is 6 seconds. We need this due to Viva is somewhat slow in its response when executiong POST request.
Future fix is to implement SQS or eq